test/EVMHost.cpp

```diff
@@ -292,6 +297,8 @@ evmc::result EVMHost::call(evmc_message const& _message) noexcept
 			result.create_address = message.destination;
 			destination.code = evmc::bytes(result.output_data, result.output_data + result.output_size);
 			destination.codehash = convertToEVMC(keccak256({result.output_data, result.output_size}));
+			if (m_evmRevision >= EVMC_BERLIN)
+				access_acount(result.create_address);
```

Comment:
According to EIP-2929 this should be done before executing initcode. So this is already done with `access_account(message.destination);`.